### DISCUSSION TITLE
Spatie Query Builder Extension

### DISCUSSION THREAD

**Discussion by layerok** [dedoc/scramble#231] [Ideas]
Just want to promote the package I developed for Scramble. @romalytvynenko How do you think, should my package be listed in the readme file of this repository?
~https://github.com/adjustmentlayer/scramble-query-builder~

UPD: the extension is moved to the new repository https://github.com/exonn-GmbH/scramble-spatie-query-builder

**Comment by romalytvynenko** [comment]
Hey @adjustmentlayer 

Thanks for the question.

And thanks for developing stuff for Scramble!

~~If I correctly understood, just installing the package won't give automatic support for the Spatie's library. Users will need to document everything manually.~~

~~This is definitely something I'd like to avoid with Scramble and hence with extensions I can recommend here. A Scramble extension I'm willing to recommend should give users the same experience as Scramble does: they install it and have 90% of their API docs ready and usable without manual documentation.~~

~~So at this point I cannot list the package here, as it will not generate all the documentation automatically and will require users to manually document it.~~

UPD: I see that I've missed the point. Let me dig a bit deeper and get back to you. 

P.S.: In some future Scramble releases I plan to improve extensions development experience, it will allow you to register all package's extensions automatically. This will help providing experience that will "just work" for end users. Please stay tuned!

**Comment by romalytvynenko** [comment]
@adjustmentlayer just checked the lib, great stuff!

I'd definitely link your package, but I'm going to create a separate page in docs instead where I'll keep links to all community-maintained packages.

If you want this to happen quicker, please feel free to make a PR to docs website repo: https://github.com/dedoc/scramble.dedoc.co

Nice job on the package! Sorry for me misunderstanding it from the first try 🤪

P.S.: Yup! This is definitely very important :)
> // todo: check if the methodName is called on QueryBuilder

**Reply by layerok** [reply]
Thanks for checking out my extension. 

> If you want this to happen quicker, ...

I can't promise, but I will try contributing to docs

By the way I think that Scramble is a great package, that deserves recognition. Thanks for sharing it on Reddit, 
without seeing this post I would use Scribe 🤮 (I'm sorry for this emoji, Scribe)

Keep it up!
